File: mau/88276.pas
var a : array[1..1000] of integer;
    n, i, j, dem : integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    for i:= 1 to n do begin
        dem:=0;
        for j:= 1 to i do
            if a[i]=a[j] then inc(dem);
        if dem=1 then write(a[i],' ');
    end;
END.

File: mau/88276.cpp
#include <iostream>
using namespace std;
int n, a[1000];
int main()
{
    int i, j, dem;
    cin>>n;
    for (i=1; i<= n; i++) cin>>a[i];
    for (i=1; i<=n; i++) {
        dem = 0;
        for (j=1; j<=i; j++)
            if (a[i]==a[j]) dem++;
        if (dem==1) cout<<a[i]<<" ";
    }
    return 0;
}
